app.py

```diff
@@ -90,13 +107,12 @@
                 ),
                 html.Div(children=[
                     dcc.Graph(id='feature-graph', style={'flex': 1}, responsive=True),
-                    html.Div(id='score-div', style={'flex-basis': '230px'}),
+                    dcc.Graph(id='accuracy-graph', style={'flex-basis': '230px'}, responsive=True, config={'displayModeBar': False}),
```

Comment:
This line is causing console errors
```suggestion
                    dcc.Graph(id='accuracy-graph', style={'flexBasis': '230px'}, responsive=True, config={'displayModeBar': False}),
```